packages/packages/commonwell-sdk/src/models/human-name.ts

```diff
@@ -28,8 +28,8 @@ export const humanNameSchema = z.object({
   use: nameUseCodesSchema.optional().nullable(),
   text: z.string().optional().nullable(),
   family: z.array(z.string()),
-  given: z.array(z.string()),
-  prefix: z.string().optional().nullable(),
+  given: z.array(z.string()).optional(),
```

Comment:
general comment: should we make all of the `.optional()` pieces `.optional().nullable()` as well, just in case?

Reply:
I'd suggest we move forward with this version and I can add it to the next PR that would replicate the `additionalInfo` solution to other CW SDK functions. 

TBH, I don't remember having `null`, but better safe than sorry when it comes to CW. :)

Sounds good?

Reply:
Discussed offline, agreed to move forward with this version